ibmcloud/README.md

```diff
@@ -91,29 +188,24 @@ As usual, you need to create `terraform.tfvars` to specify parameter values. The
 ibmcloud_api_key = "<your API key>"
 ssh_key_name = "<your SSH key name>"
 cluster_name = "<cluster name>"
+primary_subnet_name = "<name of your primary subnet>" OR primary_subnet_id = "<ID of your primary subnet>"
+primary_security_group_name = "<name of your primary security group>" OR primary_security_group_id = "<ID of your primary security group>"
+vpc_name = "<name of your VPC>" OR vpc_id = "<ID of your VPC>"
```

Comment:
Optional/idea: Should we provide the user more guidance of where to find these variables e.g. point out that the ids should be printed as output variables at the end of the **Create VPC** section?

Reply:
I added a section under this part saying if the input variables can come from another Terraform configuration where they come from and where to find them, if the user used the Terraform configurations to create the resources (the user might have done it manually).

I did similar sections for the `podvm-build`, `start-cloud-api-adaptor` and `run-nginx-demo` Terraform configurations as well, so these sections should read similarly.